wrap_semop.c

```diff
@@ -0,0 +1,14 @@
+#include <unistd.h>
+#include <asm/unistd.h>
+#include <sys/syscall.h>
+
+/* Forward declare to avoid compiler warning */
+struct sembuf;
```

Comment:
It's just a pointer, so it doesn't matter *too* much, but we can get an equivalent thing by:

```
#include <linux/sem.h>
```

Reply:
Include header in 1b4a7c64509494168eddb88321ca62bc6f6cd2a8